ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CreateStreamCommand.java

```diff
@@ -37,7 +38,12 @@ public CreateStreamCommand(
   @Override
   public DdlCommandResult run(final MetaStore metaStore) {
     if (registerTopicCommand != null) {
-      registerTopicCommand.run(metaStore);
+      try {
+        registerTopicCommand.run(metaStore);
+      } catch (KsqlException e) {
+        throw new KsqlException(String.format("Cannot create stream '%s': %s",
```

Comment:
Can you add `e` to the new exception please, e.g. `throw new KsqlException(newMessage, e);`

Reply:
Done